### ISSUE TITLE
Could not find generated BPL file with imports.

### ISSUE THREAD

**Issue by michael-emmi** [SRI-CSL/solidity#30]
This should be mostly self-explanatory: when imports are used, it looks like `solc-verify.py` generates a Boogie file named for the first-imported source module, rather than the source module to be verified, and fails to find it.

```
$ cat Imported.sol && cat Test.sol && solc-verify Test.sol --output out --verbose ; ls out && cat out/Imported.sol.bpl 
// Imported.sol
pragma solidity ^0.5.0;
// Test.sol
pragma solidity ^0.5.0;

import "Imported.sol";

contract Test {
    int x;

    /**
     * @notice modifies x
     * @notice x == __verifier_old_int(x) + 1
     */
    function test() public {
        x ++;
    }
}Solc command: /Users/mje/Code/solidity-boogie/build/solc/solc  --boogie Test.sol -o out --overwrite --boogie-arith int
----- Compiler output -----
Warning: This is a pre-release compiler version, please do not use it in production.

======= Converting to Boogie IVL =======

======= Imported.sol =======

======= Test.sol =======

---------------------------
Using z3 at /usr/local/bin/z3
Verifier command: mono /Users/mje/Code/boogie/Binaries/Boogie.exe out/Test.sol.bpl /nologo /doModSetAnalysis /errorTrace:0 /useArrayTheory /trace /z3exe:/usr/local/bin/z3
Error while running verifier, details:
[TRACE] Using prover: /usr/local/bin/z3
Parsing out/Test.sol.bpl
Error opening file "out/Test.sol.bpl": Could not find file "/Users/mje/Code/smart-contract-explorer/out/Test.sol.bpl"

Imported.sol.bpl
// Global declarations and definitions related to the address type
type address_t = int;
var __balance: [address_t]int;
type string_t;
var __now: int;
var __block__number: int;
// 
// ------- Source: Imported.sol -------
// Pragma: solidity^0.5.0
// 
// ------- Source: Test.sol -------
// Pragma: solidity^0.5.0
// 
// ------- Contract: Test -------
// 
// State variable: x : int256
var {:sourceloc "Test.sol", 7, 5} {:message "x"} x#4: [address_t]int;
// 
// Function: test : function ()
procedure {:sourceloc "Test.sol", 13, 5} {:message "Test::test"} test#11(__this: address_t, __msg_sender: address_t, __msg_value: int)
	ensures {:sourceloc "Test.sol", 13, 5} {:message "Function might modify 'x' illegally"} (x#4[__this] == (if true then x#4[__this] else old(x#4[__this])));

{
	var inc#8: int;
	inc#8 := x#4[__this];
	x#4 := x#4[__this := (x#4[__this] + 1)];
	$return0:
}

// 
// Default constructor
procedure {:sourceloc "Test.sol", 6, 1} {:message "Test::[implicit_constructor]"} __constructor#12(__this: address_t, __msg_sender: address_t, __msg_value: int)
{
	__balance := __balance[__this := 0];
	x#4 := x#4[__this := 0];
}
```

**Comment by hajduakos** [comment]
Fixed in f6a43ae1dc93f7c9c6d794fbd4f7175f3bd6d9cf, @michael-emmi check if it works for you as well

**Comment by michael-emmi** [comment]
👍
